src/Template/Reviews/of.ctp

```diff
@@ -34,8 +34,19 @@ $categories = array(
     'outdated' => ['keyboard_arrow_right', __("Outdated reviews")]
 );
 
-if (empty($correctnessLabel) || !in_array($correctnessLabel, $categories)) {
-    $category = $categories['all'][1];
+$categoriesWithLang = [
+    'ok' => __('Sentences in {language} marked as "OK"'),
+    'unsure' => __('Sentences in {language} marked as "unsure"'),
+    'not-ok' => __('Sentences in {language} marked as "not OK"'),
+    'all' => __('All sentences in {language}'),
+    'outdated' => __('Outdated reviews for sentences in {language}')
+];
+
+if ($lang) {
+    $category = format(
+        $categoriesWithLang[$correctnessLabel],
```

Comment:
This produces an `Undefined index` notice if `$correctnessLabel` is set to an unexpected value (e.g. http://localhost:8080/fra/reviews/of/contributor/blah)

Reply:
We should probably redirect to `all` directly from the controller if the `$correctnessLabel` parameter is invalid, instead of validating it in the view.